### ISSUE TITLE
CMM MODEL REQUEST

### ISSUE THREAD

**Issue by KarenHdez24** [NeurasysResearch/Knee_X-Rays_CNN#2]
Good afternoon, 
Would it be possible to have access to your CNN model?

Thanks in advance.

**Comment by Tommy-Ngx** [comment]
You can find the CNN model by use the model.summary from TF. it will show all layers of model!

**Comment by Tommy-Ngx** [comment]
This is the basic model, you can make a similar one and train in the google colab to test your models. You can also download some other pre-trained to test their performance!
Model: "sequential"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
conv2d (Conv2D)              (None, 254, 254, 32)      320       
_________________________________________________________________
max_pooling2d (MaxPooling2D) (None, 127, 127, 32)      0         
_________________________________________________________________
conv2d_1 (Conv2D)            (None, 125, 125, 64)      18496     
_________________________________________________________________
max_pooling2d_1 (MaxPooling2 (None, 62, 62, 64)        0         
_________________________________________________________________
conv2d_2 (Conv2D)            (None, 60, 60, 256)       147712    
_________________________________________________________________
max_pooling2d_2 (MaxPooling2 (None, 30, 30, 256)       0         
_________________________________________________________________
flatten (Flatten)            (None, 230400)            0         
_________________________________________________________________
dropout (Dropout)            (None, 230400)            0         
_________________________________________________________________
dense (Dense)                (None, 128)               29491328  
_________________________________________________________________
dense_1 (Dense)              (None, 64)                8256      
_________________________________________________________________
dense_2 (Dense)              (None, 5)                 325       
=================================================================
Total params: 29,666,437
Trainable params: 29,666,437
Non-trainable params: 0

**Comment by NeurasysResearch** [comment]
Hola,claro

Qué necesitas exactamente ?


saludos

**Comment by KarenHdez24** [comment]
En tu modelo /content/Knee_XRays_CNN.h5
he visto que accuracy alcanza un valor de 98%, pero este no es compartido, sera possible tener acceso a el?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by NeurasysResearch** [comment]
Hola,


El accuracy se obtiene en la etapa de evaluación y se puede ver al activar
la siguiente celda:

[image: image.png]

También al activar la celda llamada new_model.summary() , puedes observar
la estructura de la red neuronal , todas las capas y parámetros que utiliza.

Espero haber aclarado tus dudas ,


Saludos
